lib/signs/utilities/updater.ex

```diff
@@ -19,43 +19,27 @@ defmodule Signs.Utilities.Updater do
         top_msg,
         bottom_msg
       ) do
-    case {Messages.same_content?(sign.current_content_top, top_msg),
-          Messages.same_content?(sign.current_content_bottom, bottom_msg)} do
-      {true, true} ->
+    case Messages.same_content?(sign.current_content_top, top_msg) and
+           Messages.same_content?(sign.current_content_bottom, bottom_msg) do
+      true ->
         sign
 
-      # update top
-      {false, true} ->
-        log_line_update(sign, top_msg, "top")
-
-        sign.sign_updater.update_single_line(
-          sign.text_id,
-          "1",
-          top_msg,
-          sign.expiration_seconds + 15,
-          :now
-        )
-
-        %{sign | current_content_top: top_msg, tick_top: sign.expiration_seconds}
-
-      # update bottom
-      {true, false} ->
-        log_line_update(sign, bottom_msg, "bottom")
-
-        sign.sign_updater.update_single_line(
-          sign.text_id,
-          "2",
-          bottom_msg,
-          sign.expiration_seconds + 15,
-          :now
-        )
-
-        %{sign | current_content_bottom: bottom_msg, tick_bottom: sign.expiration_seconds}
-
-      # update both
-      {false, false} ->
-        log_line_update(sign, top_msg, "top")
-        log_line_update(sign, bottom_msg, "bottom")
+      _ ->
+        top_msg =
+          if Messages.same_content?(sign.current_content_top, top_msg) do
```

Comment:
nit: Since these `same_content?` calls are non-trivial and central to the logic here, it might be nice do do them once at the beginning and store the results in variables, so it's clear at a glance that the same values are being used further down.

Reply:
Ah yeah, that's a good call!